src/gui/transferlistwidget.cpp

```diff
@@ -690,8 +690,16 @@ void TransferListWidget::setSelectedFirstLastPiecePrio(const bool enabled) const
         torrent->setFirstLastPiecePriority(enabled);
 }
 
-void TransferListWidget::setSelectedAutoTMMEnabled(const bool enabled) const
+void TransferListWidget::setSelectedAutoTMMEnabled(const bool enabled)
 {
+    if (enabled)
+    {
+        const QMessageBox::StandardButton btn = QMessageBox::question(this, tr("Enable automatic torrent management")
+                , tr("Are you sure you want to enable Automatic Torrent Management for the selected torrent(s)?")
```

Comment:
Maybe add a warning that it can cause relocating?

Reply:
Added "They may be relocated". I couldn't find a pronoun neutral way to say it (to include the case of only one torrent) without making it sound... forced.

Reply:
>"This may cause relocation." ?

Doesn't sound right to me.

Reply:
> > "This may cause relocation." ?
> 
> Doesn't sound right to me.

Okay, forget it.